scripts/modal.js

```diff
@@ -13,10 +13,10 @@ function getAll(selector) {
     return ret;
 }
 
-if (!localStorage.getItem('soccersim-shownchanges')) {
-    localStorage.setItem('soccersim-shownchanges', true);
-    openModal('notification-modal');
-}
+// if (!localStorage.getItem('soccersim-shownchanges')) {

```

Comment:
No longer showing the modal with the change log?

Reply:
No more automatically opening the modal